src/olympia/addons/management/commands/process_addons.py

```diff
@@ -145,7 +146,19 @@ def get_recalc_needed_filters():
         'qs': [Q(migrated_from_lwt__isnull=False,
                  type=amo.ADDON_STATICTHEME, status=amo.STATUS_APPROVED,
                  addonapprovalscounter__last_content_review__isnull=True)]
-    }
+    },
+    'delete_obsolete_addons': {
+        'method': delete_addons,
+        'qs': [
+            Q(type__in=(amo.ADDON_THEME,
+                        amo.ADDON_LPADDON,
+                        amo.ADDON_PLUGIN,
+                        9,  # ADDON_PERSONA,
```

Comment:
nit: Personally I'd use `_ADDON_PERSONA` here and even add `_ADDON_WEBAPP` to our base constants and use that as well.

Reply:
yeah I was just being lazy because `_`'d properties aren't exposed in `amo.`

Reply:
yeah I was just being lazy because `_`'d properties aren't exposed in `amo.`